app/controllers/system/admin/users_controller.rb

```diff
@@ -53,4 +57,10 @@ def user_params
   def search_param
     params.permit(:search)[:search]
   end
+
+  def paginate_page_params
```

Comment:
I think you dont need to repeat this. Can place this function in ApplicationPaginationConcern and declare it once there. It can then be re-used in all controllers already